UI/window-basic-status-bar.hpp

```diff
@@ -6,6 +6,17 @@
 #include <util/platform.h>
 #include <obs.h>
 
+#define OPT_DYN_BITRATE "DynamicBitrate"
+
+enum dynamicBitrateState {
+	BITRATE_EQUAL_INITIAL_BITRATE,
+	BITRATE_SWITCHING_DOWN,
+	BITRATE_SWITCHING_STATIONARY,//means bitrate is lower than initial bitrate so that
```

Comment:
Nitpick:  Comment ends in punctuation, but does not seem to start as a sentence.